cjson/sbom_cJSON.yml

```diff
@@ -0,0 +1,8 @@
+name: cJSON
+version: "1.7.19"
+cpe: cpe:2.3:a:cjson_project:cjson:{}:*:*:*:*:*:*:*
+cpe: cpe:2.3:a:davegamble:cjson:{}:*:*:*:*:*:*:*
```

Comment:
@fhrbata Please check if we can specify multiple cpe records here. At-least in the scan, I see only latest cpe record after this change:

```
  "records": [
    {
      "vulnerable": "NO",
      "pkg_name": "cJSON",
      "pkg_version": "1.7.19",
      "cve_id": "",
      "cvss_base_score": "",
      "cvss_base_severity": "",
      "cvss_version": "",
      "cvss_vector_string": "",
      "cpe": "cpe:2.3:a:davegamble:cjson:1.7.19:*:*:*:*:*:*:*",
      "keyword": "",
      "cve_link": "",
      "cve_desc": "",
      "exclude_reason": "",
      "status": ""
    },
```

Reply:
Syntax issue - now fixed